PENDING_RELEASE_NOTES.md

```diff
@@ -3,6 +3,8 @@
 _Date TBD_
 
 ### New Features
+* Respond with a 401 error when the user's token has expired
```

Comment:
This is behind a flag and should go in FLAGGED_RELEASE_NOTES.md similar to how it's done in the UI. See chat in Teams for reference.